src/RpcClient.ts

```diff
@@ -52,6 +53,9 @@ export abstract class RpcClient {
             }
 
             console.debug('RpcClient RECEIVE', data);
+            if (persistMessage) {
+                window.sessionStorage.setItem(`response-${data.id}`, JSONUtils.stringify(message));
+            }
```

Comment:
I personally like it better to have the parameter indicating whether to store a request or not. I don't think it would be obvious why the _receive return value would indicate it. I also like it somewhat better to have it done in the base class.

Reply:
The storing has been moved to the child class.

Reply:
Could have added me as a co-author for that commit too ;)